lib/services/messenger/translators/translator-scaffold.ts

```diff
@@ -84,6 +85,26 @@ export abstract class TranslatorScaffold implements Translator {
 		}
 	}
 
+	protected static extractThreadId(service: string, messages: string[]): Promise<IdentifyThreadResponse> {
```

Comment:
Lets add some tests for this ;) https://github.com/resin-io-modules/resin-procbots/pull/339